__test__/commentCounter.test.js

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable no-undef */
+/**
+ * @jest-environment jsdom
+ */
+
+import commentCount from "../src/modules/commentCounter.js";
+/* eslint-disable */
+describe("Counters", () => {
+  it("Comments counters", () => {
+    document.body.innerHTML = `<div class="left">
+  <p class="eachScore">
+    2022-09-01
+    <span>Ritika:</span>
+  </p>
+  <span>
+    <p class="numberSc">Great</p>
+  </span>
+</div>`;
+    const count = document.querySelectorAll(".left");
+    expect(commentCount(count)).toBe(1);
+  });
+});
```

Comment:
- [ ] Hello, team. You did an excellent job of addressing some of the previous reviewer's changes 👏. However, when I run the test, I get the same error that the previous reviewer mentioned.
![Screenshot from 2023-03-24 23-27-22](https://user-images.githubusercontent.com/89228743/227661852-9bbefb57-4e95-40a0-9d8b-c4c0beff1458.png)


The problem is not that he lacks the necessary setup to run the test, but that the code is running in an environment where there is no "document" object available, which is a node environment rather than a browser. To resolve the issue, we can use Node.js tools like JSDOM to simulate a browser environment.

So I think updating you code as follow will fix the issue: 

```
const { JSDOM } = require('jsdom');
const dom = new JSDOM(`<!DOCTYPE html><body></body></html>`);
global.window = dom.window;
global.document = window.document;

describe("Counters", () => {
  it("Comments counters", () => {
    document.body.innerHTML = `<div class="left">
      <p class="eachScore">
        2022-09-01
        <span>Ritika:</span>
      </p>
      <span>
        <p class="numberSc">Great</p>
      </span>
    </div>`;
    const count = document.querySelectorAll(".left");
    expect(commentCount(count)).toBe(1);
  });
});
```

Reply:
Thankyou  @ridwanediallo  for clearing our doubts.. It worked
thanks 😃😄